tracing-flame/src/error.rs

```diff
@@ -0,0 +1,11 @@
+use displaydoc::Display;
+use std::path::PathBuf;
+
+#[derive(Display, Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
Let's not publicly expose an enum error type, since users can match against it exhaustively. Can we hide this behind a struct?

Reply:
Should `tracing-flame` abide by the 1.39 MSRV? If not, it can make use of the #[non_exhaustive] attribute.